variables.tf

```diff
@@ -12,6 +12,30 @@ variable "name" {
   description = "The name to give the IBM Event Notification instance created by this module."
 }
 
+variable "destination_name" {
+  type        = string
+  description = "The Destintion name."
```

Comment:
The name to give the IBM Cloud Object Storage destination which will be created for storage of failed delivery events.